rsoxs/Functions/energyscancore.py

```diff
@@ -1062,7 +1062,7 @@ def flyer_scan_energy(scan_params, sigs=[], md={},locked=True,polarization=0):
         metadata
 
     """
-    detectors = [Beamstop_WAXS_int, Beamstop_SAXS_int, Izero_Mesh_int, Sample_TEY_int]
+    detectors = [Beamstop_WAXS_int, Beamstop_SAXS_int, DM7_Photodiode_int, Izero_Mesh_int, Sample_TEY_int]
```

Comment:
here is where we might want to pair down the number of things we actually fly... this is what's causing the huge delay when we end a scan or RE.stop()  I think adding the exact channels we want would be better...

Reply:
like only one of the beamstops or photodiode is every actually used, and often for transmission we don't want sample_tey, but for TEY we don't want any diodes...

Reply:
I didn't change anything here yet other than updating the signal name.  I need to trace the code more to figure out how to identify which beamstop is actually being used and then to set that as the desired detector.  Does it make sense to pull the other changes for now (assuming they all look okay) and then having this as a new pull request later?

Reply:
for sure, just pointing out if there is a longer delay, this might be why